rllib/agents/ppo/appo_tf_policy.py

```diff
@@ -444,4 +428,5 @@ def setup_late_mixins(policy: Policy, obs_space: gym.spaces.Space,
         LearningRateSchedule, KLCoeffMixin, TargetNetworkMixin,
         ValueNetworkMixin
     ],
+    view_requirements_fn=view_requirements_fn,
```

Comment:
We could actually move this into `before_loss_init` now that we have this for all policies. Then, we really don't need the view_requirements_fn in the policy templates/EagerPolicy.